src/llv8.cc

```diff
@@ -57,14 +58,62 @@ void LLV8::Load(SBTarget target) {
   types.Assign(target, &common);
 }
 
+Error::Error(bool failed, const char* format, ...) {
+  failed_ = failed;
+  char tmp[MSG_BUF_LENGTH];
+  va_list arglist;
+  va_start(arglist, format);
+  vsnprintf(tmp, sizeof(tmp), format, arglist);
+  va_end(arglist);
+  msg_ = tmp;
+}
+
+
+bool IsDebugMode() {
+  char* var = getenv("LLNODE_DEBUG");
+  if (var == nullptr) return false;
+
+  return strlen(var) != 0;
```

Comment:
Consider caching this at load time. `getenv()` does a linear search, it's not cheap with big envs.